lib/lighthouse/benefits_claims/configuration.rb

```diff
@@ -83,19 +95,27 @@ def get_access_token?
       !use_mocks? || Settings.betamocks.recording
     end
 
-    def access_token
-      token_service.get_token if get_access_token?
+    def access_token(lighthouse_client_id, lighthouse_rsa_key_path, options = {})
+      if get_access_token?
+        token_service(
+          lighthouse_client_id,
+          lighthouse_rsa_key_path,
+          options[:aud_claim_url],
+          options[:host]
+        ).get_token(options[:auth_params])
+      end
     end
 
     ##
     # @return [BenefitsClaims::AccessToken::Service] Service used to generate access tokens.
     #
-    def token_service
-      url = "#{settings.host}/#{TOKEN_PATH}"
-      token = settings.access_token
+    def token_service(lighthouse_client_id, lighthouse_rsa_key_path, aud_claim_url = nil, host = nil)
+      host ||= base_path(host)
+      url = "#{host}/#{TOKEN_PATH}"
+      aud_claim_url ||= settings.aud_claim_url
```

Comment:
we need to default to the access_token value in the settings if the `aud_claim_url` is not sent in.
something like
```
aud_claim_url ||= settings.aud_claim_url || settings.access_token.aud_claim_url
```